src/main/java/com/authy/api/Token.java

```diff
@@ -74,4 +74,6 @@ public String toXML() {
 		
 		return map;
 	}
+
+  public String toJSON(){ return ""; }
```

Comment:
it is because the space is with tabs instead of single spaces I'll re indent the file.